local-dev/scripts/finalize.js

```diff
@@ -0,0 +1,77 @@
+var fs = require('fs-extra');
+var path = require('path');
+
+var buildPath = path.join(__dirname, '../build');
+var deployPath = __dirname.replace('/local-dev/scripts', '/docs');
+
+// files to be ignored when finalizing build into /docs directory
+var blessedFiles = ['.gitignore', 'CNAME', 'status.yml'];
+
+// make sure /build directory exists
+fs.exists(buildPath, (exists) => {
+    'use strict';
+
+    if (exists) {
+        // clear out /docs directory, ignoring directories and blessed files
+        fs.readdir(deployPath, (err, items) => {
+            // iterate over each item in deployPath
+            for (let item of items) {
+                // get stats on each item
+                fs.stat(deployPath + '/' + item, (err, stats) => {
+                    // if no error and item is a file and file is *not* blessed, remove it
+                    if (!err && stats && stats.isFile() && blessedFiles.indexOf(item) === -1) {
+                        fs.remove(deployPath + '/' + item, err => {
+                            if (!err) {
+                                console.log('🗑 Removed ' + item + '!');
```

Comment:
Fancy icons in my terminal. Maybe a bit too hip for my taste but I'll let you have it :)